docs/BR.md

```diff
@@ -2030,136 +3025,61 @@ If the signing key is P-521, the signature MUST use ECDSA with SHA-512. When enc
+The following requirements apply to all Certificates listed in [Section 7.1.2](#712-certificate-content-and-extensions). Specifically, this includes Technically Constrained Non-TLS Subordinate CA Certificates, as defined in [Section 7.1.2.3](#7123-technically-constrained-non-tls-subordinate-ca-certificate-profile), but does not include certificates issued by such CA Certificates, as they are out of scope of these Baseline Requirements.
 
-For every valid Certification Path (as defined by RFC 5280, Section 6):
+For every valid Certification Path (as defined by [RFC 5280, Section 6](https://tools.ietf.org/html/rfc5280#section-6)):
 
 * For each Certificate in the Certification Path, the encoded content of the Issuer Distinguished Name field of a Certificate SHALL be byte-for-byte identical with the encoded form of the Subject Distinguished Name field of the Issuing CA certificate.
-* For each CA Certificate in the Certification Path, the encoded content of the Subject Distinguished Name field of a Certificate SHALL be byte-for-byte identical among all Certificates whose Subject Distinguished Names can be compared as equal according to RFC 5280, Section 7.1, and including expired and revoked Certificates.
-
-#### 7.1.4.2 Subject Information - Subscriber Certificates
+* For each CA Certificate in the Certification Path, the encoded content of the Subject Distinguished Name field of a Certificate SHALL be byte-for-byte identical among all Certificates whose Subject Distinguished Names can be compared as equal according to [RFC 5280, Section 7.1](https://tools.ietf.org/html/rfc5280#section-7.1), and including expired and revoked Certificates.
 
-By issuing the Certificate, the CA represents that it followed the procedure set forth in its Certificate Policy and/or Certification Practice Statement to verify that, as of the Certificate's issuance date, all of the Subject Information was accurate. CAs SHALL NOT include a Domain Name or IP Address in a Subject attribute except as specified in [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) or [Section 3.2.2.5](#3225-authentication-for-an-ip-address).
+Effective 2022-11-01, when encoding a `Name`, the CA SHALL ensure that:
 
-Subject attributes MUST NOT contain only metadata such as '.', '-', and ' ' (i.e. space) characters, and/or any other indication that the value is absent, incomplete, or not applicable.
+  * Each `Name` MUST contain an `RDNSequence`.
+  * Each `RelativeDistinguishedName` MUST contain exactly one `AttributeTypeAndValue`.
+  * Each `RelativeDistinguishedName`, if present, is encoded within the `RDNSequence` in the order that it appears in [Section 7.1.4.2](#7142-subject-attribute-encoding).
```

Comment:
How would fields need to be handled that are not in the current list? E.g. subject:businessCategory or subject:jurisdictionLocalityName from EVG? I'm also interested to know if there's a source for this requirement, or what's the driver for the ordering?

Reply:
You mean, where would these be sorted?

These are requirements derived from the semantics of RFC 5280 and X.509. A DN is hierarchical in semantic naming, and an RDN with multiple elements is saying that these names are semantically equivalent hierarchically.

For the name forms listed, they are not semantically equivalent (e.g. a `countryName` is not the same hierarchy as a `localityName` / interchangeable with), and there is a semantic hierarchy.

These are already existing logical requirements, just being made explicit.

Reply:
Yes, I wonder about the order of fields not in this list. 

I understand the hierarchy to order logic, but is the order defined in Section 7.1.4.2 based on an  existing specification, or how did we come to this ordering?

Reply:
It is based on the definitions within X.509 and X.520, given these fields are generally geographical in nature.

That said, there’s definitely flexibility here to get us closer to consistency among CAs, which is a key point of profiling, so if there are changes and concerns, it’s totally appropriate to highlight.

Prior relevant art includes [RFC 2377](https://datatracker.ietf.org/doc/html/rfc2377), [RFC 1218](https://www.rfc-editor.org/rfc/rfc1218.html), and [RFC 1255](https://www.rfc-editor.org/rfc/rfc1255)